### DISCUSSION TITLE
"Pin" a game at the top of profile page

### DISCUSSION THREAD

**Discussion by Alarju** [RetroAchievements/RAWeb#1093] [Ideas]
Just to make it clear, i'm not talking about the badges or the Completion Progress list, but rather the more detailed game progression that shows on "Last 5 games played"

I have some favorite games that I would like to fix on top of my profile page, think that would be interesting if users could choose which games they want to show, be it a mastered game or even just a non-mastered game where they take pride of the collected achievements. Unfortanetely, I have no idea how possible it could be, so I don't have any suggestions on how to do it =/

**Comment by TeddyWestside22** [comment]
With the latest discussions, it seems the site is trying to move towards better highlighting of a players accomplishments in a game, that doesn't require full mastery. So instead of seeing only 10/20 achievements, you might have a badge for beating the game. That's the tier talk, anyway. To this topic though, I've made a mock-up of a pinned game. I'm not sure if this is where (or what) you meant exactly...
![image](https://user-images.githubusercontent.com/32810074/185097101-022b7861-7672-4a98-9f45-7f2ddd66c474.png)

My concern would be that it pushes the Last 5 games played down  - and especially so if it's more than one game you would want to highlight in this manner. When I view someones profile, I generally want to find out at a glance what they've recently been playing and how they've been doing. 

Personally, I'd like to see a user showcase implemented, where the user themselves selects achievements from any of their games and they're displayed in a section in the right container. Something like so: 
![image](https://user-images.githubusercontent.com/32810074/185095762-a54e6a6d-6672-4aeb-a012-74fd9b7dbd37.png)
It feels a bit more tidy.

**Reply by Alarju** [reply]
Yeah that's exactly what I meant!!! That would be really cool.

And I really liked your idea of a showcase session, I would just like to add something. Maybe merging the Featured Game idea on the first image with the showcase on the second, like, idk, maybe a link over Showcase that leads to a page with "Featured Games"? The point is being able to see a more detailed progression over the game that person wants to show, since it's a game progression that holds more significance to the player, that's why I thought about the model in "Last 5 games played", cause it already shows not just how much achievements the user earned, but which ones they did, in what order and how much points he earned with them, I for one find that pretty awesome!

I'm sorry if i'm  suggesting too much btw, it's just that I don't know how much work would be needed to do it, so if it seens too troublesome or impractible I understand.
